Gameplay/entities/player/PlayerController.cpp

```diff
@@ -887,7 +894,11 @@ void Hachiko::Scripting::PlayerController::RegisterHit(float damage_received, bo
 	_invulnerability_time_remaining = _invulnerability_time;
 	_combat_stats->ReceiveDamage(damage_received);
 	UpdateHealthBar();
-	game_object->ChangeEmissiveColor(float4(255, 255, 255, 255), 0.3f, true);
+
+	if (_geo != nullptr)
+	{
+		_geo->ChangeEmissiveColor(float4(255, 255, 255, 255), 0.3f, true);
```

Comment:
The name of this variable was changed for some reason? 
I don't understand the first time very well what it was >.<

Reply:
It is a new variable, with the geometry of the player so when we change the color the dash does not get effected